src/app/assets/css/maevsi.css

```diff
@@ -49,25 +49,55 @@
     }
   }
 
+  --default-font-family: var(--font-manrope);
   --font-manrope:
     'Manrope Variable', ui-sans-serif, system-ui, sans-serif,
     'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
-  --default-font-family: var(--font-manrope);
+  --color-accent-fancy: #4743b1;
+  --color-background-bright: var(
+    --color-white
+  ); /* TODO: replace with new design variables */
+
+  --color-background-brighten: var(
+    --color-zinc-200
+  ); /* TODO: replace with new design variables */
+
+  --color-background-card-bright: #fafafc;
+  --color-background-dark: var(
+    --color-zinc-800
+  ); /* TODO: replace with new design variables */
+
+  --color-background-darken: var(
+    --color-zinc-700
+  ); /* TODO: replace with new design variables */
+
+  --color-border-accent-line: #e2e1f4;
+  --color-critic-text: #8c1818;
+  --color-link-bright: var(
+    --color-blue-400
+  ); /* TODO: replace with new design variables */
+
+  --color-link-dark: var(
+    --color-blue-600
+  ); /* TODO: replace with new design variables */
+
+  --color-shadow-card: 0 1px 2px 0 rgb(0 0 0 / 5%);
```

Comment:
The value doesn't look like a color only. Also, isn't this the same as the existing `shadow-xs`?